assets/js/index.js

```diff
@@ -13,24 +10,58 @@ class App {
     this.appContainer = container;
     this.mobileNavIcon = this.appContainer.querySelectorAll(App.selectors.dataMobileNavIcon);
     this.mobileNav = this.appContainer.querySelector(App.selectors.dataMobileMenu);
+    this.page = this.appContainer.querySelectorAll(App.selectors.dataPage);
+
+    if (window.innerWidth <= 1024) {
+      this.addMobileMenuEventListener();
+    }
   }
 
   _bindEvents() {
+    window.addEventListener('resize', this.setMobileNavOnMobileView)
+  }
+
+  setMobileNavOnMobileView = () => {
+    if (window.innerWidth > 1024) {
+      this.removeMobileMenuEventListener();
+    } else {
+      this.addMobileMenuEventListener();
+    }
+  }
+
+  addMobileMenuEventListener = () => {
+    this.page.forEach((element) => {
+      element.addEventListener('click', this.closeMobileMenu)
+    })
+
     this.mobileNavIcon.forEach(icon => {
-      icon.addEventListener('click', () => {
-        this.toggleMobileMenu();
-      })
+      icon.addEventListener('click', this.toggleMobileMenu)
     })
   }
 
-  toggleMobileMenu() {
+  removeMobileMenuEventListener = () => {
+    this.page.forEach((element) => {
```

Comment:
what exactly is `element`? should it just be `page`?

Reply:
It is the element of the page. 
There is only one page so I didn't want to make ```this.page``` to be ```this.pages``` and make each element to be ```page```. If it is confusing, I think I can name it ```pageElement```. What do you think about that name?
@mokaymokay